src/module.c

```diff
@@ -4566,6 +4568,9 @@ int RM_UnblockClient(RedisModuleBlockedClient *bc, void *privdata) {
         if (bc->unblocked) return REDISMODULE_OK;
         if (bc->client) moduleBlockedClientTimedOut(bc->client);
     }
+    else {
```

Comment:
styling: `else` should be in the same line with the closing bracket in that case

Reply:
Solved in the last commit